src/components/login/LogIn.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { login } from '../../apis/apiSevices';
+import { TextField } from '@material-ui/core';
+import { useFormik } from "formik";
+import { initialValues, basicSchema } from "../../utilities";
+import { verifiedUser } from '../../redux/reducers/loginSlice';
+import { useDispatch } from 'react-redux';
+import './LogIn.css'
+
+function LogIn() {
+
+    const dispatch=useDispatch();
+
+    const formikData = useFormik({
+        initialValues,
+        onSubmit: async (values, actions) => {
+          await new Promise((data) => setTimeout(data, 500));
+          try{
+           const user= await login(username,password);
+           dispatch(verifiedUser(user))
+          }catch(error){
+          }
+          actions.resetForm();
+        },
+        validationSchema: basicSchema,
+      });
+  return (
+    <div className="signupContainer">
+      <h1>Log In</h1>
+      <form autoComplete="off" onSubmit={formikData.handleSubmit}>
```

Comment:
create a common component for this 
and wrap login and signup component inside that component using props.children